spec.html

```diff
@@ -21708,58 +21708,58 @@ <h1>Cyclic Module Records</h1>
           </table>
         </emu-table>
 
-        <emu-clause id="sec-moduledeclarationinstantiation">
-          <h1>Instantiate ( ) Concrete Method</h1>
+        <emu-clause id="sec-moduledeclarationlinking">
```

Comment:
@ljharb Will this result in links to the previous fragment not working? If so, do we care?

Reply:
Could add `oldids="sec-moduledeclarationinstantiation"`

Reply:
Good point; Added a couple oldids.